### DISCUSSION TITLE
Article in German Computer journal c't about lesspipe

### DISCUSSION THREAD

**Discussion by pboettch** [wofr06/lesspipe.sh#150] [General]
I was amazed to see that they produced a small article about lesspipe in the German Computer Journal c't. 

https://www.heise.de/select/ct/2024/7/2404615532594903230

What I saw it, I felt very nostalgic.

**Comment by wofr06** [comment]
Hi Patrick, nice to get a sign from you. Yes, this small article was certainly created, as last year I wrote an article in 
[Linux Magazin](https://www.linux-magazin.de/ausgaben/2022/07/lesspipe-2-0/).
It was a small side product when I released the first version of my code more than 25 years ago. Meanwhile in my opinion it is a quite sophisticated piece of code and very different from the first [version.](url). Even I find it rather useful in my daily work under Linux ;-)
